lib/cmd/fh3/admin/users/listdevices.js

```diff
@@ -0,0 +1,29 @@
+/* globals i18n */
+var common = require("../../../../common");
+var fhreq = require("../../../../utils/request");
+
+//TODO: Check it still been required and working - Was not possible to found data related to it
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1135)